src/services/auth.js

```diff
@@ -0,0 +1,21 @@
+// TODO: Use typedi or similar
+const Account = require("../models/Account");
+
+class AuthService {
+    constructor() {
```

Comment:
Empty constructor is not required

Reply:
This had the purpose of setting up dependency injection using something like `typedi`, but was stopped in the interim due to other things being more relevant at the time.

Reply:
However, it can be removed without any issue